contracts/epochs/FlowEpoch.cdc

```diff
@@ -437,8 +447,6 @@ pub contract FlowEpoch {
                 FlowEpoch.borrowDKGAdmin().forceEndDKG()
             }
 
```

Comment:
When we enable auto-rewards on Mainnet, how are rewards paid out during weeks where we have a spork?

Reply:
I assume we'll do it manually